src/server/routes/units.js

```diff
@@ -104,9 +106,9 @@ router.post('/edit', async (req, res) => {
 			await unit.update(conn);
 		} catch (err) {
 			log.error('Failed to edit unit', err);
-			res.status(500).json({ message: 'Unable to edit units.', err });
+			failure(res, 500, 'Unable to edit units ' + err.toString());
 		}
-		res.status(200).json({ message: `Successfully edited units ${req.body.id}` });
+		success(res, `Successfully edited units ${req.body.id}`);
```

Comment:
This has a similar issue to conversions in that is returns the id. However, here we can use req.body.identifier instead. Could you make that change (or I can)?